apps/dashboard/src/app/team/[team_slug]/(team)/page.tsx

```diff
@@ -31,3 +33,33 @@ export default async function Page(props: {
     </div>
   );
 }
+
+async function getProjectsWithTotalWallets(
```

Comment:
isn't this going to slow down the page load quite significantly?

Reply:
That my concern as well - but it seems to be pretty quick

Reply:
what about with projects that have 100k+ wallets in them?

Reply:
It's preprocessed up to the day. So you'll be aggregating as many rows as there are days in the time period.